style.css

```diff
@@ -100,3 +100,291 @@ html {
 .contact-me li {
   list-style: none;
 }
+
+/* work-section */
+
+.work-section {
+  display: grid;
+  grid-template-rows: repeat(4, 100%);
+  grid-template-columns: 100%;
+  justify-items: center;
+  padding-bottom: 114px;
+  width: 100%;
+  background-color: #f5f5f5;
+}
```

Comment:
Kindly fix the grid following the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as the previous reviewer said. The first project in the work section is pushing the other project. Please fix that. Keep rocking 💪